rules/linux/privilege_escalation_kworker_uid_elevation.toml

```diff
@@ -57,13 +57,14 @@ tags = [
         "Use Case: Threat Detection",
         "Tactic: Privilege Escalation",
         "Tactic: Defense Evasion",
-        "Data Source: Elastic Defend"
+        "Data Source: Elastic Defend",
+        "Data Source: Elastic Endgame"
         ]
 timestamp_override = "event.ingested"
 type = "eql"
 query = '''
-process where host.os.type == "linux" and event.action == "session_id_change" and event.type == "change" and
-process.name : "kworker*" and user.id == "0"
+process where host.os.type == "linux" and event.action == "session_id_change" and process.name : "kworker*" and
```

Comment:
Is this event.action the same across Endgame and Defend?

Reply:
Yes. 
<img width="745" alt="image" src="https://github.com/elastic/detection-rules/assets/78494512/5cd926f0-3d52-464b-b431-f7f11c280d81">